### ISSUE TITLE
Right click on node does not hide open node options

### ISSUE THREAD

**Issue by BridgeAR** [MaibornWolff/codecharta#1161]
# Bug

## Expected Behavior

GIVEN a right click on a node in the map and releasing the cursor (the NODE options are visible)

WHEN clicking with the right mouse button on parts of the map without releasing the cursor

THEN the originally open NODE options are hidden

## Actual Behavior

THEN the open NODE options are kept open

## Steps to Reproduce the Problem

1. Open a map
1. Click with the right mouse button on a node and release the button
1. Click on another node with the right mouse button
1. The node option is not hidden

## Specifications
- is released in [online-demo](https://maibornwolff.github.io/codecharta/visualization/app/index.html?file=codecharta.cc.json&file=codecharta_analysis.cc.json): yes
- CodeCharta Version: 1.53.0
- OS: All
- Browser: All

**Comment by BridgeAR** [comment]
This is related to #1027

**Comment by NearW** [comment]
> This is related to #1027

Absolutely. Seems like a small bug sneaked in. I guess, I just tried to hide the context menu by pressing the left mouse button.
